src/lesson3/Task23.java

```diff
@@ -0,0 +1,16 @@
+package lesson3;
```

Comment:
Mistakes:
1) </div> should be </p>, not <p>
2) <div class='green-text'> not processing

Reply:
corrected